rebar.config

```diff
@@ -1,33 +1,29 @@
 %%-*- mode: erlang -*-
-{sub_dirs, ["rel"]}.
-
 {cover_enabled, true}.
 
 {erl_opts, [debug_info, fail_on_warning,
             {platform_define, "^[0-9]+", namespaced_types},
-            {i, "./deps/dns/include"},
-            {i, "../dns/include"},
             {parse_transform, lager_transform}]}.
 
-{deps, [
-        {parse_trans, ".*", {git, "git://github.com/uwiger/parse_trans.git", {branch, "master"}}},
-        {lager, ".*", {git, "git://github.com/basho/lager.git", {branch, "master"}}},
-        {recon, ".*", {git, "git://github.com/ferd/recon.git", {branch, "master"}}},
-        {folsom, ".*", {git, "git://github.com/folsom-project/folsom.git", {branch, "master"}}},
-        {poolboy, ".*", {git, "git://github.com/devinus/poolboy.git", {branch, "master"}}},
-        {jsx, ".*", {git, "git://github.com/talentdeficit/jsx.git", {branch, "master"}}},
+{deps, [lager,
+        recon,
+        folsom,
+        jsx,
         {dns, ".*", {git, "git://github.com/dnsimple/dns_erlang.git", {branch, "master"}}},
-        {parse_xfrm_utils, ".*", {git, "git://github.com/sargun/parse_xfrm_utils.git", {branch, "master"}}},
-        {proper, ".*", {git, "https://github.com/manopapad/proper.git", {branch, "master"}}},
-        {iso8601, ".*", {git, "https://github.com/erlsci/iso8601.git", {tag, "1.2.2"}}}
+        iso8601
 ]}.
 
+{profiles, [{test, [{deps, [proper]}]}]}.
+
+{shell, [{apps, [erldns]},
+         {config, "erldns.config.example"}]}.
```

Comment:
Any objections to changing this to "erldns.config" rather than using the example config?

Reply:
Should I rename the file? This is based on what you have in `run.sh` which uses the example config.

Reply:
No, just use `erldns.config` here, as it was in run.sh: https://github.com/dnsimple/erldns/pull/90/files#diff-0a369498a5a8db3ac8fa606b544c9810L36

Reply:
Ooh, so the idea is that the user copy it to `erldns.config` first?

Reply:
Exactly.

Reply:
If you want I can merge and then make this change post merge.

Reply:
I just updated it.

Reply:
👍